### DISCUSSION TITLE
Add --policy option to moonshine:resource command

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1220] [Ideas]
Currently, MoonShine offers separate commands for generating resources (`moonshine:resource`) and policies (`moonshine:policy`). To streamline the development process, it would be beneficial to integrate policy generation directly into the resource creation command.

Proposed functionality:

- Add a `--policy` flag to the `moonshine:resource` command
- When used, automatically execute the `moonshine:policy` command for the resource
- Set `protected bool $withPolicy = true;` in the generated resource class

This integration would save developers time and ensure that resources and their corresponding policies are created consistently.

**Comment by lee-to** [comment]
quite a simple task, maybe you can do it yourself?
